docs/content/design/communication.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Communication"
+date: 2018-02-11T15:57:56-05:00
+---
+
+## Communication flow
+
+This is the story of a time long ago. A time of myth and legend when the ancient Gods were petty and cruel and they plagued build with irreproducibility.
+Only one project dared to challenge their power...Athens. Athens possessed a strength the world had never seen. Filling its cache.
+
+### Clean plate
+
+At the beginning, there's theoretical state when a cache of proxy and registry is empty.
+
+When User makes a request at this ancient time, it works as described on the flow below.
+
+- User contacts Athens asking for module M in version v1.0
```

Comment:
Should we refer to the proxy as Athens? It might cause confusion.

ln#18,24 also